### PR TITLE
Get opaque-ke/rust/cxx/JSI setup working with opaque client functions

### PR DESCRIPTION
**Pull request by steos**

### CODE REVIEW

File: example/ios/OpaqueExample.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
Comment (nikgraf) [inline example/ios/OpaqueExample.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: why was this one necessary or why was it generated?

File: example/ios/OpaqueExample.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings
Comment (nikgraf) [inline example/ios/OpaqueExample.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:1]: why was this one necessary or why was it generated?

**Review by nikgraf — APPROVED** [review]

File: example/ios/OpaqueExample.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
Comment (steos) [reply example/ios/OpaqueExample.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: This is something generated by xcode and supposedly it's good to commit this, at least according to this SO answer https://stackoverflow.com/questions/50367896/ideworkspacechecks-plist-file-suddenly-appear-after-updated-xcode, that's why I included it.

> Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace’s shared data, to store the state of necessary workspace checks. Committing this file to source control will prevent unnecessary rerunning of those checks for each user opening the workspace.

File: example/ios/OpaqueExample.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings
Comment (steos) [reply example/ios/OpaqueExample.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:1]: This was also generated by xcode but I'm not sure about this, I didn't really commit this one intentionally.

**Pull request merged**